src/DatePicker/DatePicker.js

```diff
@@ -309,6 +311,28 @@ class DatePicker extends Component {
         this.validateDates(this.props.onBlur);
     };
 
+    _showTodayHeader = () => {
+        const { todayAction: { label: todayLabel, type: todayType } } = this.props;
+        return todayType === 'navigate'
+                && todayLabel
+                && typeof todayLabel === 'string'
+                && todayLabel.trim().length > 0;
+    }
+
+    _showTodayFooter = () => {
+        const { enableRangeSelection, todayAction: { label: todayLabel, type: todayType } } = this.props;
+        return todayType === 'select'
+                && !enableRangeSelection
+                && isEnabledDate(moment(), this.props)
```

Comment:
Why do we not show the footer button if
```js
&& !enableRangeSelection
&& isEnabledDate(moment(), this.props)
```
but we don't consider these things for the header button?

Reply:
Because clicking the footer today button selects today's date and closes the popover, I think this would not be a valid interaction if a date range is to be selected or if today's date is disabled. Also, since the header today button only performs navigation we can show it regardless. What do you think?